### PR TITLE
Add TS types for css-jss packages

### PR DESCRIPTION
**Pull request by chrisands**
## Corresponding issue (if exists):
#1246 

## What would you like to add/fix?
Add typescript types for css-jss package

### CODE REVIEW

File: packages/css-jss/src/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+export type Style = {} | null | void | string
```
Comment (kof) [inline packages/css-jss/src/index.d.ts]: you should be able to use https://github.com/cssinjs/jss/blob/master/packages/jss/src/index.d.ts#L4

Comment (chrisands) [reply packages/css-jss/src/index.d.ts]: Thank you, updated code

File: packages/css-jss/src/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import {Style} from 'jss'
+
+export {Style}
+
+export type StyleArg = Style | Array<Style>
+
+export type Css = (...args: StyleArg[]) => string
```
Comment (HenriBeck) [inline packages/css-jss/src/index.d.ts]: The definitions for `createCss` and the default export are missing here. These definitions will not work by it's own

Comment (chrisands) [reply packages/css-jss/src/index.d.ts]: Done

File: packages/css-jss/src/index.d.ts [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import {Jss, Style} from 'jss'
```
Comment (HenriBeck) [inline packages/css-jss/src/index.d.ts]:
```suggestion
import {Jss, JssStyle} from 'jss'
```

I think this is named `JssStyle` now

Comment (chrisands) [reply packages/css-jss/src/index.d.ts]: Rebased to master branch and updated imports

**Comment by kof** [comment]
thanks @chrisands and @HenriBeck

**Pull request merged**